src/sql/engine/px/ob_dfo.h

```diff
@@ -182,6 +182,26 @@ struct ObP2PDhMapInfo
   TO_STRING_KV(K_(p2p_sequence_ids), K_(target_addrs));
 };
 
+struct ObQcRecordData {
+  OB_UNIS_VERSION(1);
+public:
+  common::ObString sql;
+  uint32_t tid;
```

Comment:
Need to add an underscore at the end for class member variables.
such as: 

```tid -> tid_```

Reply:
> Need to add an underscore at the end for class member variables. such as:
> 
> `tid -> tid_`

done